webhook_server/app.py

```
@@ -170,49 +193,77 @@ def healthcheck() -> dict[str, Any]:
         except Exception as e:
             _logger.exception(f"{log_context} Error in background task: {e}")
 
     try:
         api: GithubWebhook = GithubWebhook(hook_data=hook_data, headers=request.headers, logger=logger)
-
         background_tasks.add_task(process_with_error_handling, _api=api, _logger=logger)
-        return {"status": requests.codes.ok, "message": "ok", "delivery headers": delivery_headers}
+
+        LOGGER.info(f"{log_context} Webhook queued for background processing")
+        return {
+            "status": requests.codes.ok,
```

Pylint (3.3.7)

[error] Instance of 'LookupDict' has no 'ok' member

(E1101)